model/clusters_mgmt/v1/cluster_api_type.model

```diff
@@ -20,4 +20,8 @@ struct ClusterAPI {
 	URL String
 	// The listening method of the API server.
 	Listening ListeningMethod
+	// The CIDR blocks that are allowed for the Cluster API server. Currently, only supported for ARO-HCP.
+	// The maximum number of CIDR blocks supported is 500. The CIDR blocks should be non-overlapping and valid.
+	// If no CIDR blocks provided all traffic is allowed to the Cluster API Server.
+	AllowedCIDRBlocks []String
```

Comment:
We should describe the following cases:

* What happens if I don't specify the attribute?
* Is an empty array a valid value? if not, document that. If it is, what is its behavior?